src/Raven.Server/Smuggler/Documents/StreamDestination.cs

```diff
@@ -733,17 +733,23 @@ public void WriteTombstone(Tombstone tombstone, SmugglerProgressBase.CountsWithL
 
                 using (tombstone)
                 {
-                    _context.Write(Writer, new DynamicJsonValue
+                    unsafe
                     {
-                        ["Key"] = tombstone.LowerId,
-                        [nameof(Tombstone.Type)] = tombstone.Type.ToString(),
-                        [nameof(Tombstone.Collection)] = tombstone.Collection,
-                        [nameof(Tombstone.Flags)] = tombstone.Flags.ToString(),
-                        [nameof(Tombstone.ChangeVector)] = tombstone.ChangeVector,
-                        [nameof(Tombstone.DeletedEtag)] = tombstone.DeletedEtag,
-                        [nameof(Tombstone.Etag)] = tombstone.Etag,
-                        [nameof(Tombstone.LastModified)] = tombstone.LastModified,
-                    });
+                        using (var escapedId = _context.GetLazyString(tombstone.LowerId.Buffer, tombstone.LowerId.Size))
+                        {
+                            _context.Write(Writer, new DynamicJsonValue
+                            {
+                                ["Key"] = escapedId,
```

Comment:
`tombstone.LowerId.ToString()` will have the same effect

Reply:
@grisha-kotler I'm worried about this change. We can probably pinpoint vast number of places where this is used in exact same way like here. Why this is a special case? Is something wrong with LowerId? Maybe it is not allocated properly? Maybe writing DJV is not handling this properly? Ping me on Monday, we need to discuss this.

Reply:
`LowerId` is used to just avoid do case insensitive searches. As such, we aren't threating that as a `LazyStringValue` (which has the escaping behavior).

That *is* something that we need to look at, but I don't think that happens everywhere. The only reason we don't have a `Name` property on tombstones is that we don't care about their casing, in almost all other cases we do preserve casing, so that shouldn't be an issue.